README.md

```diff
@@ -2,45 +2,154 @@
+
+You can reference this [vcpkg Quick Start](https://github.com/microsoft/vcpkg#quick-start-windows) for more details.
+
+### Building your Application
+
+In order to use the SDK installed via vcpkg with CMake, you can use the toolchain file from vcpkg:
 
-### Client: New Releases
+```cmd
+> cmake -B [build directory] -S . -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg repo]/scripts/buildsystems/vcpkg.cmake
+> cmake --build [build directory]
+```
 
-New wave of packages that we are announcing as **GA** and several that are currently releasing in **beta**. These libraries follow the [Azure SDK Design Guidelines for C++](https://azure.github.io/azure-sdk/cpp_introduction.html) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features at [Azure::Core](https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/core/azure-core/README.md).
+The **entry point** for most scenarios when using the SDK will be a top-level client type corresponding to the Azure service you want to talk to. For example, sending requests to blob storage can be done via the `Azure::Storage::Blobs::BlobClient` API.
 
-These new client libraries can be identified by the naming used for their folder, package, and namespace. Each will start with `azure`, followed by the service category, and then the name of the service. For example `azure-storage-blobs`.
+All the Azure C++ SDK headers needed to be included are located within the `<azure>` folder, with sub-folders corresponding to each service. Similarly, all types and APIs can be found within the `Azure::` namespace. For example, to use functionality form `Azure::Core`, include the following header at the beginning of your application `#include <azure/core.hpp>`.
+
+Here's an example application to help you get started:
+
+```C++
+#include <iostream>
+#include <azure/core.hpp>
+#include <azure/storage/blobs.hpp>
+
+using namespace Azure::Storage::Blobs;
+
+std::string GetConnectionString()
+{
+  return "get connection string from azure portal/environment";
+}
+
+int main()
+{
+  std::string containerName = "testcontainer";
+  std::string blobName = "sample-blob";
+
+  try
+  {
+    auto client = BlobClient::CreateFromConnectionString(
+        GetConnectionString(), containerName, blobName);
+  }
+  catch (const Azure::Core::RequestFailedException& e)
+  {
+    std::cout << e.what() << std::endl;
+    return 1;
+  }
+  return 0;
+}
+```
+
+#### Visual Studio - CMakeSettings.json
+
+When building your application via Visual Studio, you can create and update a `CMakeSettings.json` file and include the following properties to let Visual Studio know where the packages are installed and which triplet needs to be used:

```

Comment:
I would be surprised by people working with the json directly instead of the json viewer from Visual Studio. ( No actions for this comment, just sharing with you)

Reply:
I was considering including a picture like this, later:
![image](https://user-images.githubusercontent.com/6527137/108920768-1a5c8480-75ea-11eb-9076-1a4c35c7046b.png)